src/py21cmfast/src/21cmFAST.h

```diff
@@ -190,6 +195,7 @@ int CreateFFTWWisdoms(struct UserParams *user_params, struct CosmoParams *cosmo_
 void Broadcast_struct_global_PS(struct UserParams *user_params, struct CosmoParams *cosmo_params);
 void Broadcast_struct_global_UF(struct UserParams *user_params, struct CosmoParams *cosmo_params);
 void Broadcast_struct_global_HF(struct UserParams *user_params, struct CosmoParams *cosmo_params, struct AstroParams *astro_params, struct FlagOptions *flag_options);
+void Broadcast_struct_global_SFRD(struct UserParams *user_params, struct CosmoParams *cosmo_params, struct AstroParams *astro_params, struct FlagOptions *flag_options);
```

Comment:
I think you should remove this function, and follow @steven-murray's suggestion of just passing the `FILTER`, `h_peak` and `log10_k_PEAK` parameters explicitly to the functions expecting them.

The idea of these functions are to broadcast everything to all functions in the respective PS (ps.c), UF (UsefulFunctions.c) and HF (heating_helper_progs.c) files. Thus this is essentially a duplication of `Broadcast_struct_global_PS`. As you only need to pass a couple of specific arguments once or twice, explicitly adding those as function arguments would be preferable.